### ISSUE TITLE
Expose ability to produce a base form of the word used by Collator

### ISSUE THREAD

**Issue by zbraniecki** [tc39/ecma402#44]
A common use case for Collator is to sort long lists of terms, and a common UX pattern for long lists of words is to group them by letters and potentially show a sidebar letter scrolling.

Collator allows for different levels of sensitivity when comparing, but that doesn't help with grouping. We can sort "żaba" into words starting with "z", but we can't identify that after removing diacritics and reducing it to a base form it is "zaba" (and that's what Collator is using with `sensitivity = base`).

Could we expose to users?

**Comment by caridy** [comment]
definitely... can you be more precise about the current operation (from the spec) that you want to expose?

**Comment by caridy** [comment]
@zbraniecki after looking at the spec, I don't see any abstract operation that does what you want (e.g.: `let value = Normalize("żaba")`), in fact, the algo `CompareStrings()` is not even described in details, but I'm not sure we can have a final normalized version since collator is about equivalence, not necessarily transformations.

If you want to champion this, we will have to discuss where those the new functionality will live on.
